### PR TITLE
use variantID/secret default UUIDs when positing

### PR DESCRIPTION
**Pull request by matzew**

### CODE REVIEW

File: cmd/server/main.go [hunk 1]
```diff
@@ -27,7 +29,11 @@ func addAndroidVariant(labels map[string]string, client *upsClient, name string)
 		payload := &androidVariant{
 			ProjectNumber: labels["projectNumber"],
 			GoogleKey:     val,
-			variant: variant{Name:name},
+			variant: variant{
+				Name:       name,
+				VariantID:  uuid.NewV4().String(),
```
Comment (pb82) [inline cmd/server/main.go]: doesn't the variant id come from the push server?

Comment (matzew) [reply cmd/server/main.go]: yes - push server can create them too

but for "reading" we need the variandID / secret - but if we dont post them, that means different struct ... not sure that is nice ..

File: cmd/server/upsClient.go [hunk 2]
```diff
@@ -46,13 +46,18 @@ func (client *upsClient) createVariant(json []byte) {
...
 	httpClient := http.Client{}
-	_, err := httpClient.Do(req)
+	resp, err := httpClient.Do(req)
 
 	if err != nil {
 		fmt.Println("UPS request error", err.Error())
+	} else {
+		defer resp.Body.Close()
+		if resp.StatusCode >= 400 {
+			fmt.Println(resp)
```
Comment (pb82) [inline cmd/server/upsClient.go]: i think we should use something like https://golang.org/pkg/log/ instead of Println

Comment (matzew) [reply cmd/server/upsClient.go]: good call - I was already wondering, w/ all the "Println" stuff

Comment (matzew) [reply cmd/server/upsClient.go]: @pb82 I will do another PR for log - to not make this one more noisy than needed, ok ?

**Pull request merged**